Sources/HipMobileUI/ViewModels/Pages/ExhibitRouteDownloadPageViewModel.cs

```diff
@@ -42,20 +43,19 @@ public ExhibitRouteDownloadPageViewModel(IDownloadable downloadable, IDownloadab
             DownloadableDescription = downloadable.Description;
 
             Message = Strings.DownloadDetails_Text_Part1 + DownloadableName + Strings.DownloadDetails_Text_Part2;
```

Comment:
Remark from the PO: you should also change the text of message as stated in the description of your task.